src/idvc/dvc_interface.py

```diff
@@ -499,7 +504,54 @@ def view_and_load_images(self):
         self.view_image()
         self.resetRegistration()
 
+
     def SelectImage(self, image_var, image, label=None, next_button=None): 
+        if self.connection_details is None:
+            return self.SelectImageLocal(image_var, image, label, next_button)
+        else:
+            return self.SelectImageRemote(image_var, image, label, next_button)
+    
+    def SelectImageRemote(self, image_var, image, label=None, next_button=None):     
+        # start the RemoteFileBrowser
+        logfile = os.path.join(os.getcwd(), '..','..',"RemoteFileDialog.log")
```

Comment:
Clearly this is a hack. Possibly we don't want to save a log file? Or we should have another temporary directory/file

Reply:
temporary would be best